src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -54,19 +55,45 @@ export class PointerTool extends BaseTool {
       sceneController.hoveredGlyph = this.sceneModel.glyphAtPoint(point);
     }
 
-    this.sceneController.sceneModel.showResizeSelection = true;
+    this.sceneController.sceneModel.showTransformationSelection = true;
+
+    const rotationHandle = this.getRotationHandle(event, sceneController.selection);
     const resizeHandle = this.getResizeHandle(event, sceneController.selection);
     if (this.sceneController.sceneModel.hoverResizeHandle != resizeHandle) {
       this.sceneController.sceneModel.hoverResizeHandle = resizeHandle;
       this.canvasController.requestUpdate();
     }
-    if (resizeHandle) {
+    if (rotationHandle) {
+      this.setCursorForRotationHandle(rotationHandle);
+    } else if (resizeHandle) {
       this.setCursorForResizeHandle(resizeHandle);
     } else {
       this.setCursor();
     }
   }
 
+  setCursorForRotationHandle(handleName) {
+    if (handleName === "bottom-left") {
```

Comment:
If you'd name the cursors according to the handle names (I mean, why not???), then the below could be simplified to:

```js
    const cursorPositions = {
      "bottom-left": "14 14",
      "bottom-right": "16 16",
      "top-left": "16 12",
      "top-right": "16 10",
      "middle-left": "20 14",
      "middle-right": "12 14",
      "top-center": "16 18",
      "bottom-center": "16 10",
    }
    this.setCursor(`url('/images/cursor-rotate-${handleName}.svg') ${cursorPositions[handleName]}, auto`);
```

Reply:
My first naming orientation was based on icon-tabler names. But I am totally with you. I'll change it in a bit.

Reply:
Thanks. Done.